src/App.js

```diff
@@ -21,8 +19,43 @@ import {ManageProcess} from "./components/process/manage/ManageProcess";
 import {ApplicationsView} from "./components/applications/ApplicationsView";
 import {applicationsAPI} from "./utils/apis/applicationsAPI";
 import Navbar from "./components/navbar/Navbar";
-import {TasksList} from "./components/tasks/TasksList";
+import {TasksList} from "./components/tasks/crud/TasksList";
 import {getTasks} from "./test/DummyValues";
+import {TaskWrapper} from "./components/tasks/TaskWrapper";
+
+const theme = createTheme({
+    status: {
+        danger: {
+            main: '#d32f2f'
```

Comment:
To kolorki heksadecymalnie?

Reply:
to jest theme, jak chcesz je definiować inaczej, tu akurat muszą być heksadecymalnie. Ale mogę wynieść do innego pliku (tak już w sumie zrobiłem), żeby w App.js już więcej gówna nie dawać.